### PR TITLE
[kf5dbusaddons] new port

### PR DESCRIPTION
**Pull request by wrobelda**
**Describe the pull request**
Adds kf5dbusaddons. 

- Which triplets are supported/not supported? Have you updated the CI baseline?
All official should be supported.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

### CODE REVIEW

**Comment by wrobelda** [comment]
@JackBoosY ready for review now.

File: ports/kf5dbusaddons/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "kf5dbusaddons",
+  "version-string": "5.81.0",
```
Comment (JackBoosY) [inline ports/kf5dbusaddons/vcpkg.json:3]:
```suggestion
  "version": "5.81.0",
```

File: ports/kf5dbusaddons/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    PREFER_NINJA
+    OPTIONS
+        -DBUILD_HTML_DOCS=OFF
+        -DBUILD_MAN_DOCS=OFF
+        -DBUILD_QTHELP_DOCS=OFF
+        -DBUILD_TESTING=OFF
+)
+
+vcpkg_install_cmake()
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/KF5DBusAddons)
```
Comment (JackBoosY) [inline ports/kf5dbusaddons/portfile.cmake:20]: Can you please switch to `vcpkg_cmake_configure`/`vcpkg_cmake_install`/`vcpkg_cmake_config_fixup` instead and add dependencies `vcpkg-cmake` + `vcpkg-cmake-config`?

File: ports/kf5dbusaddons/portfile.cmake [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+vcpkg_copy_pdbs()
+
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin" "${CURRENT_PACKAGES_DIR}/debug/bin")
+elseif(VCPKG_TARGET_IS_WINDOWS)
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin/kquitapp5.exe" "${CURRENT_PACKAGES_DIR}/debug/bin/kquitapp5.exe")
+endif()
```
Comment (JackBoosY) [inline ports/kf5dbusaddons/portfile.cmake:28]: Seems they are some tools, can you please use `vcpkg_copy_tools` instead or add an option to disable them?

File: ports/kf5dbusaddons/portfile.cmake [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin/data)
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/bin/data)
```
Comment (JackBoosY) [inline ports/kf5dbusaddons/portfile.cmake:31]: This folder should be installed in _share/${PORT}_.

File: ports/kf5dbusaddons/portfile.cmake [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/etc)
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/etc)
```
Comment (JackBoosY) [inline ports/kf5dbusaddons/portfile.cmake:35]: This folder should be installed in _share/${PORT}_.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Comment by JackBoosY** [comment]
Pinging @wrobelda for response. Is work still being done for this PR?

File: ports/kf5dbusaddons/portfile.cmake (hunk 3 above)
Comment (wrobelda) [reply ports/kf5dbusaddons/portfile.cmake]: kquitapp5 is a non-dev tool allowing to quit an arbitrary, dbus-compatible app. No need to keep it for vcpkg purposes.

**Comment by wrobelda** [comment]
> Pinging @wrobelda for response. Is work still being done for this PR?

All fixed. Thanks for the review!

**Review by JackBoosY — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
okey, cool, thanks @wrobelda :)

**Pull request merged**